charts/gridproxy/values.yaml

```diff
@@ -11,22 +11,10 @@ image:
   # tag: "v1.7.0-rc1"
 
   env:          
-    - name: "MNEMONICS"
-      value: ""
     - name: "SERVER_PORT"
       value: ""
     - name: "EXPLORER"
```

Comment:
What is that used for? also the docs says EXPLORER_URL not EXPLORER

Reply:
this is not used /not exist. I removed that key from the values.yaml